src/cascadia/TerminalSettingsModel/ActionArgs.h

```diff
@@ -474,7 +544,7 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
             if (otherAsUs)
             {
                 return otherAsUs->_SplitStyle == _SplitStyle &&
-                       (otherAsUs->_TerminalArgs ? otherAsUs->_TerminalArgs.Equals(_TerminalArgs) :
+                       (otherAsUs->_TerminalArgs ? otherAsUs->_TerminalArgs.Equals(TerminalArgs()) :

```

Comment:
Why does this Equals have to use the fully realized value for TerminalArgs() when it isn't even an optional? Be careful here!

Reply:
oh huh, that was the only place I did that. Good catch!